CPackLists.txt

```diff
@@ -18,16 +18,18 @@
  #           COMPONENT headers)
 
 
+   SET(CMAKE_INSTALL_LIBDIR lib CACHE PATH "Output dir for libraries")
```

Comment:
please put a conditional here, or in the CMakeLists.txt file so that if it is on windows it does not go through the code below. 

Currently this will only work on non-windows systems but the setup as it is will break the cmake setup on Windows